web-animations-1/Overview.bs

```diff
@@ -924,6 +929,32 @@ matching condition from below:
 
 </div>
 
+### The current phase of an animation ### {#the-current-phase-of-an-animation}
+
+<a>Animations</a> provide a <a>timeline phase</a> to their <a>associated
+effect</a> called the animation's <dfn>current phase</dfn>.
+
+The <a>current phase</a> is calculated from the first matching condition from
+below:
+
+<div class="switch">
+
+:   If the animation's <a>hold phase</a> is <a lt="unresolved">resolved</a>,
```

Comment:
(This, and a number of other places, will need to be updated if we stop using the "unresolved" term for this.)